src/Layout/PresentationMenu/PresentationSettings/PresentationSettings.tsx

```diff
@@ -183,7 +183,7 @@ export class PresentationSettingsMenu extends React.Component<PresentationSettin
         return (
             <div className="columns">
                 <div className="columns">
-                    <div className="column">
+                    <div className="column" style={{minWidth: "10.5rem"}}>
```

Comment:
This cuts off the the top sliver of the upper-most checkboxes in some cases. It's also not recommended to use "style" to define things like this. It should be part of a class if it's going to be included (but it is broken, so needs fixing).